<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDlg</class>
 <widget class="QWidget" name="SettingDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>配置</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTreeWidget" name="configTree">
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="configExpand">
           <property name="text">
            <string>全部展开</string>
           </property>
           <property name="icon">
            <iconset resource="Resource.qrc">
             <normaloff>:/images/Resources/images/expand.ico</normaloff>:/images/Resources/images/expand.ico</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="configAdd">
           <property name="text">
            <string>添加节点</string>
           </property>
           <property name="icon">
            <iconset resource="Resource.qrc">
             <normaloff>:/images/Resources/images/add.ico</normaloff>:/images/Resources/images/add.ico</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="configDel">
           <property name="text">
            <string>删除节点</string>
           </property>
           <property name="icon">
            <iconset resource="Resource.qrc">
             <normaloff>:/images/Resources/images/del.ico</normaloff>:/images/Resources/images/del.ico</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="configSave">
           <property name="text">
            <string>保存数据</string>
           </property>
           <property name="icon">
            <iconset resource="Resource.qrc">
             <normaloff>:/images/Resources/images/save.ico</normaloff>:/images/Resources/images/save.ico</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="configExit">
           <property name="text">
            <string>退出</string>
           </property>
           <property name="icon">
            <iconset resource="Resource.qrc">
             <normaloff>:/images/Resources/images/treeExit.ico</normaloff>:/images/Resources/images/treeExit.ico</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>CAN</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTableWidget" name="canTable">
         <column>
          <property name="text">
           <string>序号</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>传输方向</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>时间标识</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>帧ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>帧格式</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>帧类型</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>数据长度</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>数据(HEX)</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="tabPosition">
          <enum>QTabWidget::South</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>基本操作</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="title">
              <string>基本操作</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>帧类型:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QLineEdit" name="canFrameId">
                <property name="text">
                 <string>000</string>
                </property>
               </widget>
              </item>
              <item row="0" column="5" colspan="2">
               <widget class="QLineEdit" name="canDataEdit">
                <property name="text">
                 <string>00 00 00 00 00 00 00 00</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>帧ID(HEX):</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="canFrameType">
                <item>
                 <property name="text">
                  <string>标准帧</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>拓展帧</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="7">
               <widget class="QPushButton" name="canBaseStop">
                <property name="text">
                 <string>停止</string>
                </property>
                <property name="icon">
                 <iconset resource="Resource.qrc">
                  <normaloff>:/images/Resources/images/del.ico</normaloff>:/images/Resources/images/del.ico</iconset>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>帧格式:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="6">
               <widget class="QLineEdit" name="canSendDelay">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="0" column="4">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>数据(HEX):</string>
                </property>
               </widget>
              </item>
              <item row="0" column="7">
               <widget class="QPushButton" name="canBaseSend">
                <property name="text">
                 <string>发送</string>
                </property>
                <property name="icon">
                 <iconset resource="Resource.qrc">
                  <normaloff>:/images/Resources/images/save.ico</normaloff>:/images/Resources/images/save.ico</iconset>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="canFrameFormat">
                <item>
                 <property name="text">
                  <string>数据帧</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>远程帧</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QLineEdit" name="canSendCount">
                <property name="text">
                 <string>1</string>
                </property>
               </widget>
              </item>
              <item row="1" column="8">
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="3" column="3">
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="2">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>发送次数:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="4" colspan="2">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>每次发送间隔(ms):</string>
                </property>
               </widget>
              </item>
              <item row="0" column="8">
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="2" column="7">
               <widget class="QPushButton" name="canStartup">
                <property name="text">
                 <string>启动</string>
                </property>
                <property name="icon">
                 <iconset resource="Resource.qrc">
                  <normaloff>:/images/Resources/images/bird.ico</normaloff>:/images/Resources/images/bird.ico</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_5">
          <attribute name="title">
           <string>高级操作</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>高级操作</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QTableWidget" name="tableWidget">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>100</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <column>
                 <property name="text">
                  <string>全选</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>帧类型</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>帧格式</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>帧ID</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>数据(HEX)</string>
                 </property>
                </column>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <layout class="QGridLayout" name="gridLayout_2">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_8">
                    <property name="text">
                     <string>发送次数:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1" colspan="2">
                   <widget class="QLineEdit" name="lineEdit"/>
                  </item>
                  <item row="1" column="0" colspan="2">
                   <widget class="QLabel" name="label_9">
                    <property name="text">
                     <string>每次发送间隔(ms):</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="2">
                   <widget class="QLineEdit" name="lineEdit_2"/>
                  </item>
                  <item row="2" column="0" colspan="2">
                   <widget class="QLabel" name="label_10">
                    <property name="text">
                     <string>每帧发送间隔(ms):</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="2">
                   <widget class="QLineEdit" name="lineEdit_3"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <item>
                   <widget class="QPushButton" name="advanceSend">
                    <property name="text">
                     <string>发送</string>
                    </property>
                    <property name="icon">
                     <iconset resource="Resource.qrc">
                      <normaloff>:/images/Resources/images/save.ico</normaloff>:/images/Resources/images/save.ico</iconset>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="advanceStop">
                    <property name="text">
                     <string>停止</string>
                    </property>
                    <property name="icon">
                     <iconset resource="Resource.qrc">
                      <normaloff>:/images/Resources/images/del.ico</normaloff>:/images/Resources/images/del.ico</iconset>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_6">
          <attribute name="title">
           <string>硬件操作</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>硬件操作</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QCheckBox" name="checkBox_6">
                <property name="text">
                 <string>连接CAN</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox">
                <property name="text">
                 <string>电源上电</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_2">
                <property name="text">
                 <string>GND</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_3">
                <property name="text">
                 <string>ACC</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_4">
                <property name="text">
                 <string>硬按键</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_5">
                <property name="text">
                 <string>转接板</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>画图</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 0, 255);</string>
         </property>
         <property name="text">
          <string>友情提示:所画图像顺序依次为:前后左右,如果画的不满意,鼠标右键撤销.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabelEx" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>720</width>
           <height>480</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="startCapture">
           <property name="text">
            <string>开始抓图</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stopCapture">
           <property name="text">
            <string>停止抓图</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveCoord">
           <property name="text">
            <string>保存坐标</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>关于</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>201</width>
         <height>131</height>
        </rect>
       </property>
       <property name="title">
        <string>版本信息</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>框架版本:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="frameVersion">
          <property name="text">
           <string>0.0.0.0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>文件版本:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="fileVersion">
          <property name="text">
           <string>0.0.0.0</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>程序版本:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="appVersion">
          <property name="text">
           <string>0.0.0.0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QLabelEx</class>
   <extends>QLabel</extends>
   <header location="global">Detection/QLabelEx.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
